src/type/__tests__/serialization-test.js

```diff
@@ -171,8 +185,8 @@ describe('Type System: Scalar coercion', () => {
       'ID cannot represent value: true',
     );
 
-    expect(() => GraphQLID.serialize(-1.1)).to.throw(
-      'ID cannot represent value: -1.1',
```

Comment:
Oh that's a great change: making the test clear the reason it fails is floating-point coercion not negative-number coercion.